libs/designsystem/src/lib/components/chart/chart-js/chart-js.service.ts

```diff
@@ -271,6 +271,30 @@ export class ChartJSService {
     );
   }
 
+  private getLabelsToApply(args: {
+    datasets: ChartDataset[];
+    type: ChartType;
+    indexAxis: string;
+    labels?: unknown[];
+  }) {
```

Comment:
Add return type, is it (string[] | null) or ? - do the same to contributing methods where weak or missing.
Consider strengthning typinping of label parameter as well.

Reply:
I've added the following type to `chart.types.ts`: 
```typescript
export type ChartLabel = string | string[]; // String[] allows for multi-line labels
```

The `unknown[]` type has now been replaced with `ChartLabel[]`. This should strengthen the typing of the parameter.  

The `ChartLabel` type has also been added to the public API of Kirby.

Reply:
Sweet :=)